data/block/blockV2.pb.go

```diff
@@ -164,10 +164,10 @@ func (m *MiniBlockReserved) GetTransactionsType() []byte {
 }
 
 type MiniBlockHeaderReserved struct {
-	ExecutionType           ProcessingType `protobuf:"varint,1,opt,name=ExecutionType,proto3,enum=proto.ProcessingType" json:"ExecutionType,omitempty"`
-	State                   MiniBlockState `protobuf:"varint,2,opt,name=State,proto3,enum=proto.MiniBlockState" json:"State,omitempty"`
-	IndexOfFirstTxProcessed int32          `protobuf:"varint,3,opt,name=IndexOfFirstTxProcessed,proto3" json:"IndexOfFirstTxProcessed,omitempty"`
-	IndexOfLastTxProcessed  int32          `protobuf:"varint,4,opt,name=IndexOfLastTxProcessed,proto3" json:"IndexOfLastTxProcessed,omitempty"`
+	ExecutionType           ProcessingType `protobuf:"varint,1,opt,name=ExecutionType,proto3,enum=proto.ProcessingType" json:"executionType,omitempty"`
+	State                   MiniBlockState `protobuf:"varint,2,opt,name=State,proto3,enum=proto.MiniBlockState" json:"state,omitempty"`
+	IndexOfFirstTxProcessed int32          `protobuf:"varint,3,opt,name=IndexOfFirstTxProcessed,proto3" json:"indexOfFirstTxProcessed,omitempty"`
```

Comment:
for all 2 indexes, I would remove the omitempty

Reply:
Removed